pkg/controller/perconaxtradbcluster/perconaxtradbcluster_controller.go

```diff
@@ -249,6 +263,20 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 		if err != nil && !errors.IsAlreadyExists(err) {
 			return fmt.Errorf("create PXC Service: %v", err)
 		}
+
+		// PodDistributedBudget for ProxySQL
+		if cr.Spec.ProxySQL.PodDisruptionBudget != nil {
```

Comment:
I believe if no section specified in config - we should apply "maxUnavailable=1" config by default

Reply:
Ok. No problem. 
Is it relevant for ProxySQL-spec only?

Reply:
for both